### ISSUE TITLE
Several question on laprune.py

### ISSUE THREAD

**Issue by lizhenstat** [alinlab/lookahead_pruning#3]
Hi, thanks for your great work and sharing of the code!
I have several questions on the code:
(1)  In  [this line](https://github.com/alinlab/lookahead_pruning/blob/master/method/laprune.py#L61), why we add the batch-norm score to the total score instead of multiply, from Equation(8) in the original paper, the scaling factor in batch norm is multiplied to the distortion score.

(2) what is [this scenario](https://github.com/alinlab/lookahead_pruning/blob/master/method/laprune.py#L82) mean? (I know the previous two cases are conv-->FC and conv-->conv)

(3) What is the parameter ["split"](https://github.com/alinlab/lookahead_pruning/blob/master/method/laprune.py#L6) ?

(4) why is the function [get_bn_weihts](https://github.com/alinlab/lookahead_pruning/blob/master/network/base_model.py#L52) calculated this way? (I notice the activation re-scaling use a sqrt version of the original activation. )

Any help would be appreciated and thanks for your time.
